starter_code/queries.md

```diff
@@ -4,80 +4,98 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+{name : Babelgum}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Query: {number_of_employees: {$gt: 5000}}
+Sort: {number_of_employees: 1}
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fileds.
 
-<!-- Your Code Goes Here -->
+Query: {founded_year: {$in: [2000, 2005]}}
```

Comment:
This query will give you the companies that were founded in 2000 or in 2005, but it's missing the ones that were founded in 2001, 02, 03, 04. You should have use `$lt(e)` and `$gt(e)`